src/vcCamera.cpp

```diff
@@ -211,6 +211,9 @@ void vcCamera_Apply(vcState *pProgramState, vcCamera *pCamera, vcCameraSettings
       // Save it back to the camera
       pCamera->position = pProgramState->worldAnchorPoint + rotation.apply(direction); // define new position
       pCamera->headingPitch = vcGIS_QuaternionToHeadingPitch(pProgramState->gis, pProgramState->camera.position, orientation);
+
+      if (pCamera->headingPitch.y > UD_PI)
```

Comment:
Should this be `while` instead of `if`? Is it possible for the value to be greater than `2 * UD_PI`?

Reply:
Shouldn't be possible in the current paths